src/components/Tag/Tag.scss

```diff
@@ -96,4 +98,46 @@ $icon-size: rem(16px);
     cursor: default;
     pointer-events: none;
   }
+
+  &.segmented {
+    margin-left: -1 * spacing(extra-tight);
+
+    &::after {
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+  }
+}
+
+.Link {
+  display: inline-grid;
+  color: var(--p-text);
+  outline: none;
+  border-radius: var(--p-border-radius-base);
+  text-decoration: none;
+  padding: spacing(extra-tight) spacing(tight);
+
+  .LinkText {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
```

Comment:
You can use:
```suggestion
    @include truncate;
```

I'm not sure this is actually working as you intended though:
![image](https://screenshot.click/16-34-z88bc-xw691.png)

Reply:
>  @include truncate;

Nice, will do that!

> I'm not sure this is actually working as you intended though:

Interesting, what browser would that be? Do you have the code for the playground?

I've been testing on Chrome, Firefox and Safari and that seems to be working fine in all of them

<details><summary>This is the Playground I'm using as provided by @chloerice, maybe we need to add something else?</summary>

```jsx
import React from 'react';
import {CustomersMinor, OrderStatusMinor} from '@shopify/polaris-icons';

import {Card, Stack, Badge, Page, Tag, Icon, Layout} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      <Layout>
        <Layout.Section secondary>
          <Card sectioned>
            <Stack>
              <Tag url="/customers?tag=VIP" onRemove={() => {}}>
                <Stack wrap={false} spacing="extraTight">
                  <Icon source={CustomersMinor} />
                  <span>VIP</span>
                </Stack>
              </Tag>
              <Tag
                url="/customers?tag=VIP"
                onRemove={() => {}}
                accessibilityLabel="VIP"
              >
                <Stack alignment="center" wrap={false} spacing="extraTight">
                  <Icon source={OrderStatusMinor} />
                  <span>3PL</span>
                  <span style={{position: 'relative', top: '-1px'}}>
                    <Badge status="attention" progress="partiallyComplete">
                      Partially fulfilled
                    </Badge>
                  </span>
                </Stack>
              </Tag>
              <Tag>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer
                at ipsum quam. Aliquam fermentum bibendum vestibulum. Vestibulum
                condimentum luctus metus, sed sagittis magna pellentesque eget.
                Duis dapibus pretium nisi, et venenatis tortor dignissim ut.
                Quisque eget lacus ac ex eleifend ultrices. Phasellus facilisis
                ex sit amet leo elementum condimentum. Ut vel maximus felis.
                Etiam eget diam eu eros blandit interdum. Sed eu metus sed justo
                aliquam iaculis ac sit amet ex. Curabitur justo magna, porttitor
                non pulvinar eu, malesuada at leo. Cras mollis consectetur eros,
                quis maximus lorem dignissim at. Proin in rhoncus massa. Vivamus
                lectus nunc, fringilla euismod risus commodo, mattis blandit
                nulla.
              </Tag>
              <Tag url="/customers?tag=VIP" onRemove={() => {}}>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer
                at ipsum quam. Aliquam fermentum bibendum vestibulum. Vestibulum
                condimentum luctus metus, sed sagittis magna pellentesque eget.
                Duis dapibus pretium nisi, et venenatis tortor dignissim ut.
                Quisque eget lacus ac ex eleifend ultrices. Phasellus facilisis
                ex sit amet leo elementum condimentum. Ut vel maximus felis.
                Etiam eget diam eu eros blandit interdum. Sed eu metus sed justo
                aliquam iaculis ac sit amet ex. Curabitur justo magna, porttitor
                non pulvinar eu, malesuada at leo. Cras mollis consectetur eros,
                quis maximus lorem dignissim at. Proin in rhoncus massa. Vivamus
                lectus nunc, fringilla euismod risus commodo, mattis blandit
                nulla.
              </Tag>
              <Tag url="/customers?tag=VIP">
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer
                at ipsum quam. Aliquam fermentum bibendum vestibulum. Vestibulum
                condimentum luctus metus, sed sagittis magna pellentesque eget.
                Duis dapibus pretium nisi, et venenatis tortor dignissim ut.
                Quisque eget lacus ac ex eleifend ultrices. Phasellus facilisis
                ex sit amet leo elementum condimentum. Ut vel maximus felis.
                Etiam eget diam eu eros blandit interdum. Sed eu metus sed justo
                aliquam iaculis ac sit amet ex. Curabitur justo magna, porttitor
                non pulvinar eu, malesuada at leo. Cras mollis consectetur eros,
                quis maximus lorem dignissim at. Proin in rhoncus massa. Vivamus
                lectus nunc, fringilla euismod risus commodo, mattis blandit
                nulla.
              </Tag>{' '}
            </Stack>
          </Card>
        </Layout.Section>
      </Layout>
    </Page>
  );
}
```

 </details>

Reply:
Ok cool this looks great 👍. I think it was the stack issue in my quick playground that got me